gradle-plugins/react/brownfield/src/main/kotlin/com/callstack/react/brownfield/artifacts/FlavorArtifact.kt

```diff
@@ -42,6 +43,8 @@ class FlavorArtifact(private val variant: LibraryVariant, private val variantTas
         val bundleProvider: TaskProvider<Task>? =
             artifactProject?.let { getBundleTaskProvider(it, variant) }
 
+        project.tasks.named("preBuild").dependsOn("${artifactProject?.path}:${bundleProvider?.name}")
```

Comment:
Only this change is relevant to the scope of the PR. The rest changes are related to linting fixtures